ui/dashboards/src/stories/decorators/WithDashboard.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { StoryFn } from '@storybook/react';
+import { DashboardProvider } from '@perses-dev/dashboards';
+
+export const WithDashboard = (Story: StoryFn) => {
```

Comment:
All of these providers are barebones, just enough to get things working. We'll likely want to expand them to be more extensible in the future, but this is a good starting place.

Reply:
👏